crates/revm/benches/burntpix/genesis_alloc.rs

```diff
@@ -0,0 +1,81 @@
+use lazy_static::lazy_static;
+use revm::primitives::{address, U256};
+use revm_precompile::{Address, Bytes, HashMap};
+use std::str::FromStr;
+
+lazy_static! {
+    pub static ref BURNTPIX_ADDRESS: Address = address!("49206861766520746f6f206d7563682074696d65");
+    pub static ref GENESIS_ALLOCS: HashMap<Address, GenesisAccount> = {
+        let mut state = HashMap::new();
+        state.insert(
+            address!("0a743ba7304efcc9e384ece9be7631e2470e401e"),
+            GenesisAccount {
+                nonce: None,
+                balance: U256::from(0),
+                code: Some(Bytes::from_str(BURNTPIX_BYTECODE_ONE).unwrap()),
+                storage: None,
+            },
+        );
+        let mut storage = HashMap::new();
+        storage.insert(
+            U256::ZERO,
+            U256::from_str("0x000000000000000000000000f529c70db0800449ebd81fbc6e4221523a989f05")
+                .unwrap(),
+        );
+        storage.insert(
+            U256::from_str("0x1").unwrap(),
+            U256::from_str("0x0000000000000000000000000a743ba7304efcc9e384ece9be7631e2470e401e")
+                .unwrap(),
+        );
+        storage.insert(
+            U256::from_str("0x2").unwrap(),
+            U256::from_str("0x000000000000000000000000c917e98213a05d271adc5d93d2fee6c1f1006f75")
+                .unwrap(),
+        );
+        state.insert(
+            *BURNTPIX_ADDRESS,
+            GenesisAccount {
+                nonce: None,
+                balance: U256::from(0),
+                code: Some(Bytes::from_str(BURNTPIX_BYTECODE_TWO).unwrap()),
+                storage: Some(storage),
+            },
+        );
+        state.insert(
+            address!("c917e98213a05d271adc5d93d2fee6c1f1006f75"),
+            GenesisAccount {
+                nonce: None,
+                balance: U256::from(0),
+                code: Some(Bytes::from_str(BURNTPIX_BYTECODE_THREE).unwrap()),
+                storage: None,
+            },
+        );
+        state.insert(
+            address!("f529c70db0800449ebd81fbc6e4221523a989f05"),
+            GenesisAccount {
+                nonce: None,
+                balance: U256::from(0),
+                code: Some(Bytes::from_str(BURNTPIX_BYTECODE_FOUR).unwrap()),
+                storage: None,
+            },
+        );
+
+        state
+    };
+}
+#[derive(Debug, Clone)]
+pub struct GenesisAccount {
+    /// The nonce of the account at genesis.
+    pub nonce: Option<u64>,
+    /// The balance of the account at genesis.
+    pub balance: U256,
+    /// The account's bytecode at genesis.
+    pub code: Option<Bytes>,
+    /// The account's storage at genesis.
+    pub storage: Option<HashMap<U256, U256>>,
+}
```

Comment:
`const BURNTPIX_BYTECODE_ONE: &str = `
can be made as `static BURNTPIX_BYTECODE_ONE : Bytes = bytes!("....");`